WooCommerce/Classes/POS/ViewModels/ItemListViewModel.swift

```diff
@@ -18,25 +18,22 @@ final class ItemListViewModel: ItemListViewModelProtocol {
         }
     }
 
-    var shouldShowHeaderBanner: Bool {
-        // The banner it's only shown when:
-        // - Loading the item list
-        // - Hasn't been already been previously dismissed
-        !isHeaderBannerDismissed && state.isLoaded
-    }
-
     private let itemProvider: POSItemProvider
     private let selectedItemSubject: PassthroughSubject<POSItem, Never> = .init()
 
     let selectedItemPublisher: AnyPublisher<POSItem, Never>
 
     var itemsPublisher: Published<[POSItem]>.Publisher { $items }
     var statePublisher: Published<ItemListViewModel.ItemListState>.Publisher { $state }
-    var isHeaderBannerDismissedPublisher: Published<Bool>.Publisher { $isHeaderBannerDismissed }
 
     init(itemProvider: POSItemProvider) {
         self.itemProvider = itemProvider
         selectedItemPublisher = selectedItemSubject.eraseToAnyPublisher()
+
+        // The banner is shown as long as it hasn't already been dismissed once:
```

Comment:
I wonder if this could be avoided, since `observeBannerDismissal` should assign value at least once when `state` changes. WDYT?